tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Logging/ILogger/DirectSubmission/DirectSubmissionLoggerProvider.cs

```diff
@@ -40,10 +39,19 @@ internal DirectSubmissionLoggerProvider(
             IExternalScopeProvider? scopeProvider)
         {
             _sink = sink;
-            _formatter = formatter;
             _minimumLogLevel = minimumLogLevel;
             _createLoggerFunc = CreateLoggerImplementation;
-            _scopeProvider = scopeProvider;
+
+#if NETCOREAPP3_1_OR_GREATER
+            if (sink is OtlpSubmissionLogSink)
+            {
+                _logEventCreator = new OtelLogEventCreator();
```

Comment:
Last thing, can you handle the scope provider the same way that you're handling it in our standard Sink? Unless you have a specific reason for not propagating scope changes, let's keep the handling consistent

```suggestion
                _logEventCreator = new OtelLogEventCreator(scopeProvider);
```

Reply:
So currently we don't make use of the scopeProvider, I tried adding it and the logs differed from OTEL so I want to connect on how we should use that data and the effect on OTEL customers.